src/main/java/com/robinpowered/sdk/RobinApi.java

```diff
@@ -83,6 +83,10 @@ public RobinApi(Credential credential) {
      * @param httpClient An HTTP client for performing network requests.
      */
     public RobinApi(Credential credential, Client httpClient) {
+        if (credential != null && credential.getValue() == null) {
```

Comment:
We're OK with `credential == null`? Trying to think if we should just enforce that it's `@NonNull`